lib/entities/HardQuoteRequest.ts

```diff
@@ -76,16 +80,24 @@ export class HardQuoteRequest {
     return utils.getAddress(this.order.info.baseOutputs[0].token);
   }
 
+  public get totalOutputAmountStart(): BigNumber {
+    let amount = BigNumber.from(0);
+    for (const output of this.order.info.baseOutputs) {
```

Comment:
this assumes all outputs are same token which is not necessarily true right?

Reply:
Assumed true anyways for now in main rfq flownsince quoters can only be for one token